timesketch.conf

```diff
@@ -36,3 +36,6 @@ ELASTIC_PORT = 9200
 # another name you can configure that here.
 SSO_ENABLED = False
 SSO_USER_ENV_VARIABLE = u'REMOTE_USER'
+
+# Upload folder for Plaso dump files to be processed.
```

Comment:
Can you expand this a bit? This is where storage files will be parked temporarily before they're inserted into elastic, right?